### DISCUSSION TITLE
مشکل در login

### DISCUSSION THREAD

**Discussion by mrhamidifar** [Gozargah/Marzban#143] [Q&A]
از امروز بعد از لاگین به داشبورد منتقل نمی‌شوم 
سرور و داگر را ریستارت کردم حل نشد

**Comment by amirinjast** [comment]
سرور رو down کن و سپس‌ دستور زیر رو وارد کن

`rm -rf /var/lib/marzban/run`

و مجددا سرور رو ران کن .
